ci/install-static-desktop-file-validate.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+set -euxo pipefail
+
+if [[ "${1:-}" == "" ]]; then
+    echo "Usage: $0 <version>"
+    exit 2
+fi
+
+version="$1"
+
+build_dir="$(mktemp -d -t desktop-file-utils-build-XXXXXX)"
+
+cleanup () {
+    if [ -d "$build_dir" ]; then
+        rm -rf "$build_dir"
+    fi
+}
+trap cleanup EXIT
+
+pushd "$build_dir"
+
+apk add glib-static glib-dev autoconf automake
```

Comment:
Please install all dependencies in the central `build-in-docker.sh`. These scripts just install things that cannot be `apk add`ed.

Reply:
Then no one knows what is installed for which reason.

Reply:
...unless we comment it